tests/general/InvalidLinkBearTest.py

```diff
@@ -154,6 +171,21 @@ def test_redirect_threshold(self):
                           invalid_file=short_url_redirect,
                           settings={'follow_redirects': 'yeah'})
 
+    def test_multiple_results_per_line(self):
+
+        file = """
```

Comment:
I am not a fan of using 'file' as a variable name, as it is a [builtin function](https://docs.python.org/2/library/functions.html?highlight=file#file) name in py2 only.